src/Controller/JobsController.php

```diff
@@ -0,0 +1,296 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use DateTime;
+use Exception;
+use App\Entity\Jobs;
+use App\Repository\JobsRepository;
+use App\Repository\CompanyRepository;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+/**
+ * JobsController
+ */
+class JobsController extends AbstractController
+{
+    private JobsRepository $jobsRepository;
+    private CompanyRepository $companyRepository;
+    
+    /**
+     * __construct
+     *
+     * @param  mixed $jobsRepository
+     * @param  mixed $companyRepository
+     * @return void
+     */
+    public function __construct(JobsRepository $jobsRepository, CompanyRepository $companyRepository)
+    {
+        $this->jobsRepository = $jobsRepository;
+        $this->companyRepository = $companyRepository;
+    }
+        
+    /**
+     * addJob
+     *
+     * @param  mixed $request
+     * @return Response
+     */
+    public function addJob(Request $request): Response
+    {
+        try {
+            $name = $request->get("name");
+            $active = (int)$request->get('active');
+            $description = $request->get("description");
+            $priority = (int)$request->get('priority');
+            $companyId = (int)$request->get('company_id');
+
+            $company =  $this->companyRepository->find($companyId);
+
+            if (is_null($company)) {
+                throw new \InvalidArgumentException('Could not find company with id: ' . $companyId);
+            }
+
+            //create new Jobs objec
+            $job = new Jobs();
+            $job->setName($name);
+            $job->setDescription($description);
+            $job->setActive($active);
+            $job->setPriority($priority);
+            $job->setCreatedAt(new DateTime());
+            $job->setCompany($company);
+
+            $savedJob = $this->jobsRepository->save($job);
+
+            return new JsonResponse(
+                [
+                    'results' => [
+                        'added' => $savedJob,
+                        'error' => false,
+                    ]
+                ]
+            );
+        } catch (Exception $e) {
+            return new JsonResponse(
+                [
+                    'results' => [
+                        'error' => true,
+                        'message' => $e->getMessage()
+                    ]
+                ]
+            );
+        }
+    }
+
+    /**
+     * getJobById
+     *
+     * @param  mixed $id
+     * @return Response
+     */
+    public function getJobById(int $id): Response
+    {
+        try {
+            $job = $this->jobsRepository->find($id);
+
+            if (null === $job) {
+                throw new \InvalidArgumentException('Could not find company with id: ' . $id);
```

Comment:
Message 'Could not find company with id' is out if the context, function does not get companies. Replace with: ''Could not find job with id''